packages/react/src/passkey.ts

```diff
@@ -0,0 +1,3 @@
+import { Utils } from "@slashid/slashid";
+
+export const runtimeSupportsPasskeys = Utils.runtimeSupportsBestPasskeyUX();
```

Comment:
This might cause trouble if executed server side. Can we make sure this either runs client side only or gets parametrised so the user agent string is passed in? Server side we can read it from the HTTP request.